Clinical-Genomics/microSALT
# Basic cgMLST implementation

# Description

End-users:
- New core-genome distance analysis (based on cgmlst.org)

## Primary function of PR

# Testing:
- `bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-microsalt-stage.sh new_definitions`
- `Select options: yes, yes, new_definitions`
- `us`
- `source activate S_microSALT`
- `export MICROSALT_CONFIG=/home/proj/dropbox/microSALT.json`
- `microSALT analyse project ACC5551` (CG matrix comparison)
- `microSALT analyse project MIC4109`
- `microSALT analyse project MIC3109`

1. Make sure results converge in accordance with 1797:10 section 3.3.4
2. Check that the runfile.sh of a single sample has the `--careful` flag present
3. Use the command `microSALT utils report MIC3109 --output .` twice; and make sure both reports are present
4. Evaluate the report for a "Responsible bioinformatician field" and confirm that it suggests it's you.

## Test results


# Sign-offs
